xcode/Subconscious/Shared/Components/Settings/GatewayURLSettingsView.swift

```diff
@@ -142,11 +152,24 @@ struct GatewayURLSettingsView: View {
                     Text("Provision Gateway")
                 },
                 footer: {
-                    switch app.state.gatewayProvisioningStatus {
-                    case let .failed(message):
-                        Text(message)
-                    default:
-                        EmptyView()
+                    VStack {
+                        if let gatewayId = app.state.gatewayId {
+                            HStack {
+                                Text("Gateway ID")
+                                    .foregroundColor(.secondary)
+                                    .bold()
+                                Text(gatewayId)
+                                    .foregroundColor(.secondary)
+                            }
+                        }
```

Comment:
Render the Gateway ID if we have it, useful for debugging & support.

<img width="361" alt="Screenshot 2023-05-29 at 12 34 45 pm" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/eb6ad5e6-ca13-4e3e-a0db-ebe9788a6a4b">